lib/response/CallResponse.js

```diff
@@ -1,13 +1,7 @@
-var Rx = require("rx/dist/rx") && require("rx/dist/rx.aggregates");
```

Comment:
This was the other area where I wanted to do some due diligence before pulling this out. We may need to go back and look at the history. I think this may have been added fairly recently to work around a build issue with WebPack/rx.aggregates. If not, this is fine to remove.